packages/cli/src/CommandsLogic.ts

```diff
@@ -307,24 +320,27 @@ export class CommandsLogic {
         console.log(`Staking relayer ${stakeValueFormatted}`,
           stake.toString() === '0' ? '' : ` (already has ${currentStakeFormatted})`)
 
-        if (options.mintToken) {
-          console.log('Minting 100 tokens')
-          const mintTx = await this.contractInteractor.erc20Token.mint(100e18.toString(), {
+        const stakingTokenContract = await this.contractInteractor._createERC20(stakingToken)
+        const tokenBalance = await stakingTokenContract.balanceOf(options.from)
+        if (tokenBalance.lt(stakeValue) && isDefaultToken) {
+          console.log(`Wrapping ${formatTokenAmount(stakeValue, 18, 'eth')}`)
+          // default token is wrapped eth, so deposit eth to make then into tokens.
```

Comment:
This is a wild assumption. At least try to view-call this before sending a transaction with value to this address. Also, obviously, this behaviour cannot be a silent default. Use a flag and default it to `false`